nesta/core/schemas/tier_1/datasets/arxiv.json

```diff
@@ -0,0 +1,28 @@
+{
+    "entity_type": "article",
```

Comment:
I would prefer if we called this `source` as I'm suggesting [here](https://github.com/nestauk/nesta/issues/215#issuecomment-636059532) so to reserve the "entity" terminology for actually entities (City, Country, Person, Org, etc) to introduce later on when we'll start using ontologies.

This would also make it easier to have documents from different `source`s in the same index, e.g. `arxlive` having `source` = `arxiv` | `biorxiv` | `medrxiv`.

Reply:
I wouldn't take this field too seriously, it will be removed in daps2 as the entity and source will be tied to fields rather than the entire dataset - and these will be automatically propagated. It's basically in here as a legacy support to HealthMosaic which required an general entity type for each dataset.

Reply:
OK